pkg/ctl/cmdutils/test_data/managed-nodegroup-with-container-runtime.yaml

```diff
@@ -0,0 +1,14 @@
+# A simple example of ClusterConfig object:
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: test-cluster-1
+  region: eu-north-1
+
+managedNodeGroups:
```

Comment:
unmanaged right?

Reply:
Nope, this is supposed to fail. It's testing that you *cannot* do this.

Reply:
aaaah i thought this file was an example.yaml 🤦‍♀️  ignore me